public/styles/_home.scss

```diff
@@ -143,8 +143,8 @@ a.featured-app-list-item {
 // Announcement banner------------------------------
 
 .announcement-banner {
-  background: #23252f;
-  border-bottom: 1px solid #191a22;
+  background: $jumbo-bg-color;
```

Comment:
I think the announcement banner should probably have more contrast. I'd put `background` to `$jumbo-bg-border` and remove the actual `border-bottom`.

Reply:
```suggestion
  background: $jumbo-bg-border;
```